core/server/data/migration/fixtures/populate.js

```diff
@@ -1,98 +0,0 @@
-// # Populate Fixtures
```

Comment:
Population happens with `knex-migrator init`.
You can find the most important pieces in the init scripts in `data/migrations/init`.